src/main/java/com/springtest/dto/UserResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package com.springtest.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
FYI: it seems Data annotation suits better instead of three separate annotations